_posts/2017-05-01-five-lessons-learned.markdown

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title:  "Five Lessons Learned During My First Year as a Technical Director"
+date:   2017-05-01 12:00:00 -0400
+categories: culture management leadership
+author: Patrick Hereford
+---
+
+Making the leap from an Engineer to a Director can be daunting. There are so
+many other responsibilities that get placed on you as a director that it can be
+overwhelming. The hardest shift, I feel, is the switch from spending most of your
+time and energy coding to solving other types of problems. Most of
+that time shifts towards higher priority responsibilities like: budgeting,
+planning, product strategy, mentoring, recruitment, and so much more.
```

Comment:
do we have a style for oxford commas? I haven't been using them, but standard journalism style omits them.